### ISSUE TITLE
IDEA: seeing reads for more than one sample without pedigree file

### ISSUE THREAD

**Issue by GruA** [brentp/jigv#18]
Hello Brent!

Thank you for creating jigv, it looks promising.
 
I work with normal-tumor matched samples as well as samples collected over time and it would be great to see them together. I believe that I can make a fake pedigree file for them, right? Pedigree format is something new for me, so I will appreciate any suggestions how to make it!

If the above will not work, it would be great to add more than one --sample and load bam/cram files for many samples as it is possible with pedigree file.

Best wishes,
Agata

**Comment by brentp** [comment]
Hi Agata, the pedigree file is optional. A single tumor-normal pair will be straight-forward. For many pairs that you want to have together, you could try making a fake pedigree file.
This use-case is probably less than optimal in jigv so I'm happy for any feedback if you give it a try.

**Comment by mhanbioinfo** [comment]
Hi Brent, I was wondering what would be the command for saving a tumor-normal pair to a single .html file? I have a tumor-normal pair VCF, and tried:

        ${SRC_DIR}/jigv \
            --sample ${NORMAL_SNAME},${TUMOR_SNAME} \
            --sites TUMOR_NORMAL.vcf.gz \
            --fasta ${FASTA} \
            --annotation ${REF_ANNOT} \
            --flank=100 \
            ${BAM_DIR}/${NORMAL_SNAME}_realigned_recalibrated.subset_for_IGV.bam \
            ${BAM_DIR}/${TUMOR_SNAME}_realigned_recalibrated.subset_for_IGV.bam \
            > ${JIGV_OUT_DIR}/${TUMOR_SNAME}.html

but this did not work.

Thanks in advance,
Ming

**Comment by brentp** [comment]
yes. let me see if I can add this easily. For now, I would just make a fake ped file with both sapmles in the same family ID.

columns are tab-separated: family_id\tsample_id\tpaternal_id\tmaternal_id\tsex\tphenotype

you can use 0 or -9 for maternal and paternal id.

**Comment by mhanbioinfo** [comment]
Ah sounds good will try ped
